grails-app/i18n/messages.properties

```diff
@@ -3481,6 +3484,22 @@ react.stockMovement.enum.AvailableItemStatus.PICKED=Picked
 react.stockMovement.enum.AvailableItemStatus.RECALLED=Recalled
 react.stockMovement.enum.AvailableItemStatus.HOLD=Hold
 react.stockMovement.enum.AvailableItemStatus.NOT_AVAILABLE=Not Available
+react.requisitionStatus.enum.CREATED=Created
```

Comment:
Can we stop duplicating these? I don't understand why the react app can just use the existing key
```
enum.RequisitionStatus.STATUS
```

Reply:
As I know we were using only those labels starting with "react" on the react side. For now, when I use our function for fetching translations, I have to pass "requisitionStatus" as an argument to fetch all of the translations that start with "react.requistionStatus...".

I am moving the translation to the backend because those translations don't work as @kchelstowski suggested in a comment. So, I am removing those duplications.